vars/fcosKola.groovy

```diff
@@ -3,6 +3,7 @@
 //    cosaDir:         string  -- cosa working directory
 //    parallel:        integer -- number of tests to run in parallel (default: 8)
 //    skipUpgrade:     boolean -- skip running `cosa kola --upgrades`
+//    skipSelfTests:   boolean -- skip running `run-kola-self-tests`
```

Comment:
How about we flip this around and have the self-test be disabled by default? So we'd call this e.g. `selfTest`.

Reply:
My reason for this is that I think the self-test isn't really useful for the majority of upstream projects.

Reply:
@jlebon Thanks again for the reviews!
Done ;)